### PR TITLE
CPDRP-848 CIP only onboarding journey

### PR DESCRIPTION
**Pull request by FriedSock**
### Context

https://dfedigital.atlassian.net/browse/CPDRP-848

### Changes proposed in this pull request

- Remove the advisory page before induction programme choice. It was decided that the information on this page is a bit superfluous as it is repeated on the next page where you actually get to make the choice. It also seemed simpler than defining a different set of copy for CIP-only schools.

- Copy updates to the induction programme choice form

- Hide the `full_induction_programme` choice for CIP-only schools.

### Testing

Added a new factory trait for cip only schools, tests for the form itself regarding the list of options.

### Review Checks
- [x] All pages have automated accessibility checks via cypress
- [x] All pages have visual tests via cypress + percy
- [x] All `School` queries are correctly scoped - `eligible` when they need to be

### How can I view this in a review app?
- Log in as an induction coordinator for a school that hasn't made any choices yet. Check that the first page you are directed to is the induction programme choice page.
- Log in as an induction coordinator for the CIP only school: cip-only-induction-tutor@example.com and compare the copy and options on the page with how it looked for a FIP eligible school.

### CODE REVIEW

File: app/models/concerns/gias_helpers.rb [hunk 1]
```diff
@@ -10,6 +10,8 @@ module GiasHelpers
     scope :in_england, -> { where("administrative_district_code ILIKE 'E%'") }
     scope :section_41, -> { where(section_41_approved: true) }
     scope :eligible, -> { currently_open.eligible_establishment_type.in_england.or(currently_open.in_england.section_41) }
+    scope :cip_only, -> { currently_open.where(school_type_code: GiasTypes::CIP_ONLY_TYPE_CODES) }
```
Comment (ethanmills) [inline app/models/concerns/gias_helpers.rb:13]: This isn't quite right. If you're a CIP_ONLY type code, but have section 41 approval and are in England then you're fully eligible, not cip only

File: db/seeds/test_data.rb [hunk 2]
```diff
@@ -98,6 +98,20 @@
   )
 end
 
+School.find_or_create_by!(urn: "1818181818") do |school|
```
Comment (ethanmills) [inline db/seeds/test_data.rb:101]: This URN is the wrong length

File: app/models/concerns/gias_helpers.rb (hunk 1 above)
Comment (ethanmills) [inline app/models/concerns/gias_helpers.rb:13]: Is there a spec for this?

Comment (FriedSock) [reply app/models/concerns/gias_helpers.rb:13]: Yeah this probably doesn't make sense as a scope in isolation as the query is going to add superfluous checks. Ill just add a test for the eligible_or_cip_only scope

**Review by ethanmills — APPROVED** [review]

**Pull request merged**